tgui/packages/tgui/interfaces/chompstation/Pda/pda_screens/pda_timeclock.tsx

```diff
@@ -10,14 +10,17 @@ import {
   Section,
 } from 'tgui/components';
 
+import { formatTime } from '../../../../format';
```

Comment:
heavily recommending to use tgui/format at that depths

Reply:
Right, that would be easier, I'll push that through in a minute.